packages/playground/client/src/index.ts

```diff
@@ -94,6 +95,7 @@ export async function startPlaygroundWeb({
 		}),
 		progressTracker
 	);
+	getLogger().addPlaygroundRequestEndListener(playground);
```

Comment:
Nitpick for iteration 2: We could import an instantiated `logger` instead of a `getLogger()` function.

```
import { logger } from '@php-wasm/logger';
logger.addPlaygroundRequestEndListener(playground);
```

Reply:
Oh! This is actually blocking: the logger should only collect all the `window` errors on playground.wordpress.net in the context of the website app. However, the `client` package is a universally reusable artifact that other websites and app can reuse, e.g. https://translate.wordpress.org/projects/wp-plugins/friends/dev/de/default/playground/. do this:
 
```js
import { startPlaygroundWeb, phpVars, setPluginProxyURL } from 'https://playground.wordpress.net/client/index.js';
```

Calling `this.collectJavaScriptErrors();` in the logger constructor would make the logger collect all the Playground-unrelated global JS errors in all the apps that merely use the client. This is why I wouldn't attach any global error collection logic at the library level but defer it to the consumer app. This way, Playground `worker-thread`, `remote/index.html` and `website` could all decide to attach global error listeners without simultaneously imposing that decision on the API consumers.